conf/symbiflow/requirements.txt

```diff
@@ -12,5 +12,5 @@ pandas
 yapf
 git+https://github.com/SymbiFlow/fasm.git#egg=fasm
 git+https://github.com/SymbiFlow/xc-fasm.git#egg=xc-fasm
-git+https://github.com/SymbiFlow/edalize.git@symbiflow#egg=edalize
+git+https://github.com/antmicro/edalize.git@quicklogic#egg=edalize
```

Comment:
This change is not required anymore as https://github.com/SymbiFlow/edalize/pull/56 is now merged

Reply:
done